CODING_STANDARD.md

```diff
@@ -1,46 +1,33 @@
 Here a few minimalistic coding rules for the CPROVER source tree.
 
 # Whitespaces
-- Use 2 spaces indent, no tabs.
-- No lines wider than 80 chars.
-  - When line is wider, do the following:
-    - Subsequent lines should be indented two more than the initial line
-    - Break after `=` if it is part of an assignment
-      - For chained calls, prefer immediately before `.`
-    - For other operators (e.g. `&&`, `+`) prefer immediately after the
-      operator
-    - For brackets, break after the bracket
-      - In the case of function calls, put each argument on a separate line if
-        they do not fit after one line break
-      - Nested function calls do not need to be broken up into separate lines
-        even if the outer function call does.
-- If a method is bigger than 50 lines, break it into parts.
-- Put matching `{ }` into the same column.
-- No spaces around operators (`=`, `+`, `==` ...) Exceptions: Spaces around
-  `&&`, `||` and `<<`
-- Space after comma (parameter lists, argument lists, ...)
-- Space after colon inside `for`
-- For pointers and references, the `*`/`&` should be attached to the variable
-  name as opposed to the type. E.g. for a pointer to an int the syntax would
-  be: `int *x;`
-- No whitespaces at end of line
-- No whitespaces in blank lines
-- Put argument lists on next line (and indent 2 spaces) if too long
-- Put parameters on separate lines (and indent 2 spaces) if too long
-- No whitespaces around colon for inheritance, put inherited into separate
-  lines in case of multiple inheritance
-- The initializer list follows the constructor without a whitespace around the
-  colon. Break line after the colon if required and indent members.
-- `if(...)`, `else`, `for(...)`, `do`, and `while(...)` are always in a
-  separate line
-- Break expressions in `if`, `for`, `while` if necessary and align them with
-  the first character following the opening parenthesis
-- Use `{}` instead of `;` for the empty statement
-- Single line blocks without `{ }` are allowed, but put braces around
-  multi-line blocks
-- Use blank lines to visually separate logically cohesive code blocks within a
-  function
-- Have a newline at the end of a file
+
+- Formatting and whitespaces are enforced using clang-format-3.8. The
+  formatting will be checked during the CI run, and style issues will be
+  displayed at this point.
```

Comment:
Can we please retain some summary of the formatting rules as plain text? As is, it's required to reverse-engineer the clang-format configuration in order to know what to do. Yet, clang-format is the ultimate formalisation, but being stripped of any guideline is a rather poor experience.

Reply:
I've left the rules as they were previously, with a couple of small alterations. Added a sentence referring to the clang-format info in COMPILING.md.